apps/client-asset-sg/src/app/app-guards.ts

```diff
@@ -5,12 +5,16 @@ import { isNotNull } from '@asset-sg/core';
 import { User } from '@asset-sg/shared/v2';
 import { Store } from '@ngrx/store';
 import { filter, map } from 'rxjs';
-
 import { AppState } from './state/app-state';
 
 export const roleGuard = (testUser: (u: User) => boolean) => {
   const store = inject(Store<AppState>);
   return store.select(fromAppShared.selectUser).pipe(filter(isNotNull), map(testUser));
 };
 
+export const viewerGuard: CanActivateFn = () => {
```

Comment:
Viewer is already used in the application as a workgroup role. Can we call this `notAnonymousGuard` or similar?

Reply:
renamed to `anonymousGuard`